site/content/articles/monitoring-multiple-openshift-clusters.adoc

```diff
@@ -0,0 +1,74 @@
+---
+title: "Monitoring Multiple Openshift Clusters"
+date: 2019-11-08T08:05:32+01:00
+authors: ["redhat-cop"]
+categories:
+  - openshift
+  - operations
+tags:
+  - research
+draft: true
+---
+= Monitoring Multiple Openshift Clusters
+Andrea Ferraresi <andrea.ferraresi@redhat.com>
+:toc: macro
+:toc-title:
+
+include::site/layouts/variables.adoc[]
+
+How to deal with monitoring of multiple Openshift Cluster: the challenges and the different approaches on scaling 
+
+toc::[]
+
+== Introduction to prometheus
+We all know Prometheus to be a very simple, very easy to use, piece of software:
+
+It is just a scraping server, this means that reads its metrics from a clear text page exposed by a web server on a specific port, metrics are generated from "exporters" which fills those pages. Once metrics are scraped are stored on disc for visualization. 
```

Comment:
this means that **it** reads its...

Full stop after port and begin a new sentence with Metrics.

Once metrics are scraped**,they** are stored....

disc or disk is one of those American Vs British English, I think disk is more common for our line of work, just flagging it but feel free to ignore as it is ok :)